crates/bitwarden-crypto/src/keys/master_key.rs

```diff
@@ -69,13 +71,17 @@ impl MasterKey {
     }
 
     /// Derives a users master key from their password, email and KDF.
-    pub fn derive(password: &[u8], email: &[u8], kdf: &Kdf) -> Result<Self> {
-        derive_kdf_key(password, email, kdf).map(Self)
+    pub fn derive(password: &SensitiveVec, email: &[u8], kdf: &Kdf) -> Result<Self> {
+        derive_kdf_key(password.expose(), email, kdf).map(Self)
```

Comment:
Are we planning on changing `derive_kdf_key` to also accept a sensitive?

Reply:
derive_kdf_key needs a bit of extra work yet, as the current impl is writing the resulting hash to a fixed size array which is causing leak problems. I'm adding some tests for it now and I'll leave it to a separate PR.